src/config/config.ts

```diff
@@ -25,6 +25,7 @@ export default {
     authType: {
       token: 'token',
       gitlabToken: 'gitlab_token',
+      gitlabCommit: 'gitlab_commit',
```

Comment:
I don't use gitLab, so sorry if this is a stupid question, but can you explain why we have this line here? Is this the same as a Project token and just confusingly named on gitlabs side?

Reply:
what we have now is the gitlab trigger token. Which can only trigger a pipeline from a repository.

the gitlab_commit is requires either project/group/personal token. And it will commit the file directly to repo.

I name it gitlab_commit because it commit.

Feel free to suggest any other name.